content/hi/docs/reference/glossary/affinity.md

```diff
@@ -0,0 +1,22 @@
+---
+title: आत्मीयता
+id: affinity
+date: 2019-01-11
+full_link: /docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity
+short_description: >
+     पॉड्स को कहां रखा जाए, यह निर्धारित करने के लिए शेड्यूलर द्वारा उपयोग किए जाने वाले नियम
+aka:
+tags:
+- fundamental
+---
+
+कुबेरनेट्स में, _affinity_ नियमों का एक सेट है जो शेड्यूलर को संकेत देता है कि पॉड्स को कहाँ रखा जाए।
+
+<!--more-->
+आत्मीयता दो प्रकार की होती है:
+* [नोड आत्मीयता](/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity)
+* [पॉड-टू-पॉड आत्मीयता](/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity)
+
+कुबेरनेट्स का उपयोग करके नियमों को परिभाषित किया गया है {{< glossary_tooltip term_id="label" text="labels">}},
```

Comment:
I think you need to translate the text string here instead of just leaving it as labels.

Reply:
Updated on the new commit. Thanks Divya